tests/WebControllerTest.php

```diff
@@ -277,4 +277,13 @@ public function testFindCustomTemplatesDirectoryNotFound() {
         $this->assertEmpty($result);
     }
 
+    public function testParseVocabularyLanguageOrder() {
+        $request = new Request($this->model);
+        $vocab = $this->model->getVocabulary('multilang');
+
+        $languageOrder = $this->webController->parseVocabularyLanguageOrder($vocab);
+        $expected = array( 'en' => 'anglais', 'fi' => 'finnois', 'sv' => 'suédois', 'de' => 'allemand' );
+        $this->assertEquals($expected, $languageOrder);
```

Comment:
This test doesn't actually verify the order of the array. `assertEquals` will only check that the array has the same keys and values, but not their order.

I suggest changing to `assertSame`, which also checks the order. You will also have adjust the expected order so that the test will pass with the given vocabulary configuration `skosmos:fallbackLanguages ( "fi" "en" "sv" "de" )`:

```php
        $expected = array( 'fi' => 'finnois', 'en' => 'anglais', 'sv' => 'suédois', 'de' => 'allemand' );
        $this->assertSame($expected, $languageOrder);
```